extension/js/common/api/email-provider/gmail/google-auth.ts

```diff
@@ -121,7 +121,11 @@ export class GoogleAuth {
     }
     const authRequest: AuthReq = { acctEmail, scopes, csrfToken: `csrf-${Api.randomFortyHexChars()}` };
     const url = GoogleAuth.apiGoogleAuthCodeUrl(authRequest);
-    const oauthWin = await windowsCreate({ url, left: 100, top: 50, height: 800, width: 550, type: 'popup' });
+    const adaptiveHeight = Math.floor((window.innerHeight || window.screen.height) * 0.9);
+    const adaptiveWidth = Math.floor((window.innerWidth || window.screen.width) * 0.4);
+    const leftOffset = Math.floor((window.screen.width / 2) - (adaptiveWidth / 2) + (window as any).screen.availLeft); // tslint:disable-line:no-unsafe-any
+    const topOffset = Math.floor((window.innerHeight / 2) - (adaptiveHeight / 2) + (window as any).screen.availTop); // tslint:disable-line:no-unsafe-any
```

Comment:
based on `screenHeight` value introduced above

Reply:
Ok, sir. I tried using the screenHeight over window.innerHeight and it works nicely.